### DISCUSSION TITLE
求教，怎么发表情包

### DISCUSSION THREAD

**Discussion by LuminSAMA** [share121/inter-knot#5357] [其他]
我囤了好多好多表情包，但是发现发帖写评论要进网页，找不到我的表情包了(｡•́︿•̀｡)

**Comment by aino2333** [comment]
站内有一个绳网常见问题＆解决方法的帖子，有问题可以搜索这个帖子来看看。应该有帮助。帖内也有可以使用米游表情包的脚本<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" >

**Reply by LuminSAMA** [reply]
感谢指路，不愧是绳网，刚发的委托就解决了<img src="https://bbs-static.miyoushe.com/static/2024/03/21/6431de7537491cb394a82e8a0ff8cf77_9185062232016213590.png" alt="丽都漫步-猫又" width="75" >
